pkgs/development/python-modules/aider-chat/default.nix

```diff
@@ -9,11 +9,8 @@
 }:
 
 let
-  python3 = python312.override {
-    self = python3;
-    packageOverrides = _: super: { tree-sitter = super.tree-sitter_0_21; };
-  };
-  version = "0.75.2";
+  python3 = python312;
```

Comment:
Does this module still only work with python 3.12? Does it support 3.12+? Or is that restriction no longer necessary with the updates to tree sitter?

Reply:
I'm not entirely sure whether it works with Python 3.13 or not, but I noticed that a related issue is still open and pinned upstream.

On the other hand, I personally prefer not to pin the Python version within Python packages. Instead, I suggest marking them as disabled when incompatible, and pinning the Python version in the application itself.